torch/CMakeLists.txt

```diff
@@ -678,7 +678,7 @@ if (BUILD_PYTHON)
       list(APPEND TORCH_PYTHON_SRCS ${TORCH_SRC_DIR}/csrc/distributed/c10d/init.cpp)
       list(APPEND TORCH_PYTHON_LINK_LIBRARIES c10d)
       list(APPEND TORCH_PYTHON_COMPILE_DEFINITIONS USE_C10D)
-      if (USE_CUDA)
+      if (USE_CUDA OR USE_ROCM)
```

Comment:
Is that unconditionally correct (I know, they do it for CUDA) or should this not be `USE_RCCL`?

Reply:
I don't think this should be USE_RCCL, because the distributed api can work with multiple backends, one of which is nccl (or rccl in our case). Others are gloo, mpi, tcp etc.

Reply:
OK